vitrina/smart_contracts/migrations/0008_remove_agreement_other_assignee_legislations.py

```diff
@@ -0,0 +1,42 @@
+# Generated by Django 4.2.23 on 2025-11-07 13:28
+
+from django.db import migrations
+
+
+def move_other_assignee_legislations_to_project(apps, schema_editor):
+    Agreement = apps.get_model("vitrina_smart_contracts", "Agreement")
+    Project = apps.get_model("vitrina_projects", "Project")
+
+    updated_projects = {}
+
+    agreements_to_update = Agreement.objects.exclude(
+        other_assignee_legislations__isnull=True,
+        other_assignee_legislations=""
+    )
```

Comment:
I think this will always be false (a.k.a nothing excluded). As one field cannot be two things at the same time.
```suggestion
    agreements_to_update = Agreement.objects.exclude(
        Q(other_assignee_legislations__isnull=True) |
        Q(other_assignee_legislations="")
    )
```

Reply:
Oopsie, nice catch! Either way, all the object would be filtered and looped through, but what you suggested is what I wanted to do, thanks:D


70e76c5